### PR TITLE
Van

### PR DESCRIPTION
**Pull request by vandalat**
my week 6 solutions

### CODE REVIEW

File: src/week_06/ArrayFindMaximum.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        int max = Integer.MIN_VALUE;
+        for (int each   :   arr) {
+            if (each>max) {
+                max=each;
+            }
+        }
+        return max;
+    }
+    public static int max1 (int[]  arr){
+        return Arrays.stream(arr).max().getAsInt();
```
Comment (srestr53) [inline src/week_06/ArrayFindMaximum.java:22]: When you have time Van , you should explain me stream method . It looks so great

File: src/week_06/PasswordValidationTaks.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                    lowerCase ++;
+                }
+                if (ch >= 33 && ch <= 47) {
+                    specialChar++;
+                }
+                if (Character.isDigit(ch)) {
+                    digit++;
+                }
+
+                if (upperCase >= 1 && lowerCase >= 1 && specialChar >= 1 && digit >= 1){
```
Comment (srestr53) [inline src/week_06/PasswordValidationTaks.java:93]: This is interesting , first time I see that approach

File: src/week_06/SumOfDigit.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+//
+
+
+
+    }
+
+    public static int sumOfDigit1(String str) {
+        int sum = 0;
+        for (int i = 0; i < str.length(); i++) {
+            if(str.charAt(i)>='0'&&str.charAt(i)<='9'){
```
Comment (srestr53) [inline src/week_06/SumOfDigit.java:24]: Impressive, you're using ASCII table like a pro

**Review by srestr53 — APPROVED** [review]
Excellent .

**Review by hayrunnisa00 — APPROVED** [review]

**Pull request merged**